cachi2/core/package_managers/yarn_classic/main.py

```diff
@@ -27,6 +28,11 @@ def _ensure_mirror_dir_exists(output_dir: RootedPath) -> None:
         prefetch_env = _get_prefetch_environment_variables(request.output_dir)
         _verify_corepack_yarn_version(path, prefetch_env)
         _fetch_dependencies(path, prefetch_env)
+        # Workspaces metadata is not used at the moment, but will
+        # eventualy be converted into components. Using a noop assertion
+        # to prevent linters from complaining.
```

Comment:
I'd say either drop the direct usage for the time being, or, if you want to make sure we don't forget about it `_ = workspaces` feels cleaner than an assertion (admittedly I have a strong bias against assertions).

Reply:
Still feels weird to have in the code like that at any point in time, but okay, not a blocker.